heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesController.java

```diff
@@ -79,8 +83,128 @@ public boolean killTopology() {
       // Disconnect to release resources
       conn.disconnect();
     }
+  }
+
+  /**
+   * Get information about a pod
+   *
+   * @return json object for pod
+   */
+  public JsonNode getBasePod(String podId) {
+
+    String podURI = String.format(
+        "%s/api/v1/namespaces/default/pods/%s",
```

Comment:
Is this path always this way for all kubernetes users? Is `default` ever something else for example?